Sources/Kitura/bodyParser/BodyParser.swift

```diff
@@ -131,9 +133,9 @@ public class BodyParser: RouterMiddleware {
 
     /// Read the body data of the request.
     ///
-    /// - Parameter with: the socket reader.
+    /// - Parameter with: The socket reader.
     /// - Throws: Socket.Error if an error occurred while reading from a socket.
-    /// - Returns: data for the body.
+    /// - Returns: Data for the body.
```

Comment:
"Data for the body." doesn't make sense to me how about something like "The body data of the request."?